### DISCUSSION TITLE
Priority in non-random mode

### DISCUSSION THREAD

**Discussion by NGC3034** [MusicPlayerDaemon/MPD#1426] [Q&A]
Possible?

**Comment by MaxKellermann** [comment]
Yes. By reordering the queue.

**Reply by NGC3034** [reply]
So would that be closest to when in MOCp I choose items with "z" for temporary override of queue and when all choosen items finished it continues to play the item next to the one when first "z" selection was made?

**Reply by MaxKellermann** [reply]
What is MOCp and what is "z"? What is a temporary override of queue? What choosen items?

**Reply by NGC3034** [reply]
MOCp - music on console player
z - hot key which adds selected song to kind of subqueue. MOCp woul play all of those selected and than continue to the queue
As I understand mpd queue is basically an sequence of songs to play. Temporary override would be "forget queue for now, play these songs first and when finished continue to queue"
Choosen items - assigned with priority number for temporary override

Guessing may be I should be asking this question on ncmpcpp forum

**Reply by MaxKellermann** [reply]
There's no such temporary override in MPD.
Yes, the queue is the list of songs to be played. If you want to play any (order of) songs, change the queue. It's that simple, nothing else, no sub-queues, no overrides. Simplicity is a feature - if you don't like MPD's simplicity, MPD is not for you, just keep using MOCp or whatever.
Asking on any ncmpcpp forum will not get you any other answer, because ncmpcpp is just a dumb frontend for MPD.

**Reply by NGC3034** [reply]
Actually I like MPD (using it with ncmpcpp and Audioward). It's just some missing handy features I used to. I understand simplicity for server side, thats why that feature should probably be part of client side
